From 017723f4bf1298e0c4c257ea38f2563b5be06bd8 Mon Sep 17 00:00:00 2001
From: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
Date: Mon, 14 Sep 2020 14:45:23 +0530
Subject: [PATCH 44/44] Pad register changes for IMX6R2 A2 module

This patch will change i.MX6R2 A2 revison module GPIO pad register value for
 - SMARC LID
 - SMARC POWER_BTN
 - SMARC SLEEP
 - PCIE_WAKE_B
 - SMARC LCD_BKLT_EN
 - SDIO card detection and write protection

Signed-off-by: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
---
 arch/arm/boot/dts/lec-imx6.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/lec-imx6.dtsi b/arch/arm/boot/dts/lec-imx6.dtsi
index c60408cd8840..833c759a6fff 100644
--- a/arch/arm/boot/dts/lec-imx6.dtsi
+++ b/arch/arm/boot/dts/lec-imx6.dtsi
@@ -921,9 +921,9 @@
 
 		pinctrl_gpio_keys: gpio_keysgrp {
 			fsl,pins = <
-				MX6QDL_PAD_GPIO_7__GPIO1_IO07           0x1b0b0 /* SMARC LID# */
-				MX6QDL_PAD_GPIO_6__GPIO1_IO06           0x1b0b0 /* SMARC POWER_BTN# */
-				MX6QDL_PAD_NANDF_D7__GPIO2_IO07         0x1b0b0 /* SMARC SLEEP# */
+				MX6QDL_PAD_GPIO_7__GPIO1_IO07           0x1A8B0 /* SMARC LID# */
+				MX6QDL_PAD_GPIO_6__GPIO1_IO06           0x1A8B0 /* SMARC POWER_BTN# */
+				MX6QDL_PAD_NANDF_D7__GPIO2_IO07         0x1A8B0 /* SMARC SLEEP# */
 				MX6QDL_PAD_GPIO_8__GPIO1_IO08           0x1b0b0 /* SMARC TEST# */
 			>;
 		};
@@ -1055,13 +1055,13 @@
 		pinctrl_pcie: pciegrp {
 			fsl,pins = <
 				MX6QDL_PAD_GPIO_17__GPIO7_IO12		0x1b0b0	/* PCIE_RST_B */
-				MX6QDL_PAD_CSI0_DATA_EN__GPIO5_IO20     0x1b0b0 /* PCIE_WAKE_B */
+				MX6QDL_PAD_CSI0_DATA_EN__GPIO5_IO20     0x1F0B0 /* PCIE_WAKE_B */
 			>;
 		};
 
 		pinctrl_pwm1: pwm1grp {
 			fsl,pins = <
-				MX6QDL_PAD_NANDF_RB0__GPIO6_IO10	0x80000000 /* SMARC LCD_BKLT_EN */
+				MX6QDL_PAD_NANDF_RB0__GPIO6_IO10	0x1A8B0 /* SMARC LCD_BKLT_EN */
 				MX6QDL_PAD_GPIO_9__PWM1_OUT		0x1b0b1 /* SMARC LCD_BKLT_PWM */
 				MX6QDL_PAD_EIM_D31__GPIO3_IO31		0x80000000 /* SMARC LCD_BKLT_EN (iMX6R2) */
 			>;
@@ -1141,9 +1141,9 @@
 				MX6QDL_PAD_SD2_DAT1__SD2_DATA1		0x17059
 				MX6QDL_PAD_SD2_DAT2__SD2_DATA2		0x17059
 				MX6QDL_PAD_SD2_DAT3__SD2_DATA3		0x17059
-				//MX6QDL_PAD_GPIO_4__SD2_CD_B		0x80000000 /* controller internal CD doesn't work */
-				MX6QDL_PAD_GPIO_4__GPIO1_IO04		0x80000000 /* SD2_CD_B: mux as GPIO instead */
-				MX6QDL_PAD_GPIO_2__SD2_WP		0x80000000
+				MX6QDL_PAD_GPIO_4__SD2_CD_B		0x1F070 /* controller internal CD doesn't work */
+				/*MX6QDL_PAD_GPIO_4__GPIO1_IO04		0x1F070*/ /* SD2_CD_B: mux as GPIO instead */
+				MX6QDL_PAD_GPIO_2__SD2_WP		0x1F070
 			>;
 		};
 
-- 
2.17.1

